cli/internal/run/run.go

```diff
@@ -484,6 +505,49 @@ func noopPersistentOptsDuringMigration(flags *pflag.FlagSet) {
 	}
 }
 
+const (
+	_graphText      = "graph"
+	_graphNoValue   = "bool|text"
```

Comment:
I think we can push people towards a filename here, and then update the usage text to give a few examples of extensions that will work.

```suggestion
	_graphNoValue   = "<output filename>"
```